### DISCUSSION TITLE
How about combining power values in a common import/export gauge?

### DISCUSSION THREAD

**Discussion by fkroepfl** [UtilitechAS/amsreader-firmware#803] [Ideas]
Main argument: Since import and export of power can never take place at the same time, you could display the current value in a common gauge without losing information and release room for other tiles in the GUI.

What do you think?

<img width="1483" alt="Bildschirmfoto 2024-06-05 um 14 42 58" src="https://github.com/UtilitechAS/amsreader-firmware/assets/6333880/89792ded-15db-49ba-b3ba-0aee4b3b458a">

**Comment by ArnieO** [comment]
Some 3-phase installations can have import and export simultaneously, on different phases. The values received from the power meter will in such situation vary depending on power meter and model. 
- Some report the net import/export (one of the values is always zero)
- Some report value both on import and export

**Reply by fkroepfl** [reply]
> * Some report the net import/export (one of the values is always zero)

Thanks for this background information.

I think that in Austria all meters provide the values in netted form, because here it is possible, for example, to connect only a single-phase battery storage system and its feed-in quantity is totally included in the total calculation regardless of consumption on the other phases.
